tests/apidocs/endpoints/projects/test-project-stats.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+
+from django.core.urlresolvers import reverse
+from django.test.client import RequestFactory
+
+from tests.apidocs.util import APIDocsTestCase
+from sentry.testutils.helpers.datetime import before_now, iso_format
+
+
+class ProjectStatsDocs(APIDocsTestCase):
+    def setUp(self):
+        organization = self.create_organization()
+        project = self.create_project(name="foo", organization=organization, teams=[])
+        self.store_event(
+            data={
+                "event_id": "a" * 32,
```

Comment:
I think it makes sense to expand out the string out here.

Reply:
@mgaeta do you mean to put "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" instead or something else?

Reply:
Sorry I did a bad global search and didn't realize this is copied code. You should leave it as is.